Sources/App/Services/DiscordWebhookService.swift

```diff
@@ -1,19 +1,30 @@
 import Vapor
 
+struct DiscordError: Error {
+    let link: String
+}
+
 // Read more about discord webhooks here
 // https://discordapp.com/developers/docs/resources/webhook#execute-webhook
 struct DiscordWebhookService: Service {
     let client: Client
     let webhookUrl: String
     let logger: Logger
     let jsonEncoder: DataEncoder
-
+    
     func postContent(_ content: String) throws -> Future<Void> {
-        let content = DiscordWebhookPayload(content: content)
+        let payload = DiscordWebhookPayload(content: content)
         return client.post(webhookUrl, headers: HTTPHeaders()) { (req) in
-            req.http.body = try jsonEncoder.encode(content).convertToHTTPBody()
-        }.map(to: Void.self) { _ in
-            self.logger.info("Posted: \(content) response")
+            req.http.body = try jsonEncoder.encode(payload).convertToHTTPBody()
+        }.map(to: Void.self) { resp in
+            let code = resp.http.status.code
+            if code >= 200 && code < 300 {
+                self.logger.info("Posted: \(content) response")
+            } else if code == 429 {
+                throw DiscordError(link: content)
```

Comment:
It might make sense to rename `DiscordError` to better indicate that it's meant to be used for rate limited stuff, otherwise this looks good